contrib/kryo-serializer/src/java/com/twitter/heron/serializer/kryo/KryoSerializer.java

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.twitter.heron.api.serializer;
+package com.twitter.heron.serializer.kryo;
```

Comment:
@billonahill It's unclear from the name of the class that this is contrib. How do you package this class then? In which artifact? This name suggests this class is a part of somehow main API, which is not. 

Anyway I don't expect you to change the way you package `contrib` in this PR, but I guess it should be somehow different from what you have right now.